2022/en/src/K05-inadequate-logging.md

```diff
@@ -34,6 +36,8 @@ Managed Kubernetes providers, including AWS, Azure, and GCP provide optional acc
 
 As outlined above, there is no shortage of logging mechanisms available within the Kubernetes ecosystem. A robust security logging architecture should not only capture relevant security events, but also be centralized in a way that is queryable, long term, and maintains integrity.
 
+![Inadequate Logging - Mitigations](/assets/images/K05-2022-mitigation.gif)
+
```

Comment:
Same here. Can you provide more details?

Reply:
This is an example Audit Log highlighting the source, destination, timestamps, and other details. We will add a note to include these details.

Reply:
The illustration now has a text up top highlighting what the sample event log is. Is that good?